tests/4.5/target_teams_distribute/test_target_teams_distribute_default_none.F90

```diff
@@ -0,0 +1,87 @@
+!===--- test_target_teams_distribute_default_none.F90-----------------------===//
+!
+! OpenMP API Version 4.5 Nov 2015
+!
+! This tests uses the default(none) clause on a target teams distribute test.
+! The test aims to validate that all values will not have default data sharing
+! attributes.
+!
+!===------------------------------------------------------------------------===//
+
+
+#include "ompvv.F90"
+
+#define N 1024
+
+PROGRAM test_target_teams_distribute_default_none
+  USE iso_fortran_env
+  USE ompvv_lib
+  USE omp_lib
+  implicit none
+  OMPVV_TEST_OFFLOADING
+
+  OMPVV_WARNING("Test only uses default(none) clause and does not")
+  OMPVV_WARNING("guarantee that the default(none) is enforced.")
+  OMPVV_TEST_VERBOSE(default_none1() .ne. 0)
+  OMPVV_TEST_VERBOSE(default_none2() .ne. 0)
+  OMPVV_REPORT_AND_RETURN()
+
+CONTAINS
+  INTEGER FUNCTION default_none1()
+    INTEGER :: a(N), b(N), c(N), d(N)
+    INTEGER :: privatized, x, y, errors
+    errors = 0
+
+    DO x = 1, N
+       a(x) = 1
+       b(x) = x
+       c(x) = 2*x
+       d(x) = 0
+    END DO
+
+    !$omp target data map(from: d(1:N)) map(to: a(1:N), b(1:N), c(1:N))
+    !$omp target teams distribute default(none) shared(a, b, c, d) &
+    !$omp& private(x, privatized)
+    DO x = 1, N
+       privatized = 0
+       DO y = 1, a(x) + b(x)
```

Comment:
what about the datasharing attribute for y? 

should x be privitized if it is the iteration variable? This I am not sure

Reply:
I decided to go ahead an privatize it.